plutus-tx-plugin/test/AsData/Budget/9.6/patternMatching.uplc.golden

```diff
@@ -3,43 +3,33 @@ program
   (\d ->
      (\cse ->
         (\lessThanInteger ->
-           (\addInteger ->
+           (\cse ->
               (\cse ->
                  (\cse ->
                     (\cse ->
-                       (\cse ->
-                          (\cse ->
-                             (\cse ->
-                                (\cse ->
-                                   (\cse ->
-                                      addInteger
-                                        (addInteger
-                                           (addInteger
-                                              (addInteger (cse cse) cse)
-                                              cse)
-                                           (force
-                                              (case
-                                                 (lessThanInteger
-                                                    (cse cse)
-                                                    (cse cse))
-                                                 [ (delay (cse cse))
-                                                 , (delay (cse cse)) ])))
-                                        (force
-                                           (case
-                                              (lessThanInteger
-                                                 (cse cse)
-                                                 (cse cse))
-                                              [ (delay (cse cse))
-                                              , (delay (cse cse)) ])))
-                                     (addInteger cse))
-                                  (addInteger cse))
-                               (addInteger cse))
-                            (addInteger cse))
-                         (case cse [(\x y z w -> x)]))
-                      (case cse [(\x y z w -> y)]))
-                   (case cse [(\x y z w -> z)]))
-                (case cse [(\x y z w -> w)]))
-             (\x y -> addInteger x y))
+                       addInteger
+                         (addInteger
+                            (addInteger
+                               (addInteger (addInteger cse cse) cse)
```

Comment:
This seems like what we expect: `addInteger cse` is work-free, and based on the comments in CSE, we don't want to CSE such things.